Source/Fuse.Reactive.Expressions/Tests/UX/UnaryOperator.Basic.ux

```diff
@@ -1,7 +1,10 @@
 <Panel ux:Class="UX.UnaryOperator.Basic">
 	<FuseTest.Data Value=" 'hi' " ux:Name="v"/>
+	<FuseTest.Data FloatValue="10" ux:Name="someFloat"/>
 	
 	<FuseTest.DudElement StringValue="{= _unJoin('ab') }" ux:Name="a"/>
 	<FuseTest.DudElement StringValue="{= _unJoin({none}) ?? 'x' }" ux:Name="b"/>
 	<FuseTest.DudElement StringValue="{= _unJoin({v}) }" ux:Name="c"/>
-</Panel>
\ No newline at end of file
+
+	<FuseTest.DudElement StringValue="{= -{someFloat} }" ux:Name="d"/>
```

Comment:
Can you use just `Value="-{someFloat}"`, or as an additional test.  To ensure it works without the escape sequence as well.

Reply:
Seems like a good idea.

Reply:
I was already rewriting the tests as this was written; see new code, it better matches existing math tests and is written like this.